### PR TITLE
Strongly type AaGuid

### PR DESCRIPTION
**Pull request by iamcarbon**
This PR updates AaGuid fields from a string to a Guid. This eliminates some allocations and the need to parse the Guid later.

Making as a draft until mds3.certinfra.fidoalliance.org goes back online and tests are green.

### CODE REVIEW

File: Src/Fido2.Models/Objects/AttestationVerificationSuccess.cs [hunk 1]
```diff
@@ -12,10 +13,14 @@ public class AttestationVerificationSuccess : AssertionVerificationResult
     public byte[] PublicKey { get; set; }
 
     public Fido2User User { get; set; }
+
     public string CredType { get; set; }
-    public System.Guid Aaguid { get; set; }
+
+    public Guid AaGuid { get; set; }
```
Comment (iamcarbon) [inline Src/Fido2.Models/Objects/AttestationVerificationSuccess.cs:19]: Breaking change: updated casing for consistency.

**Comment by iamcarbon** [comment]
mds3.certinfra.fidoalliance.org is currently offline -- causing the tests to fail.

**Comment by iamcarbon** [comment]
@abergs @aseigler Ready for review.

NOTE: The WriteGuidBigEndian extension is in place to simplify the logic in AttestedCredentialData, but waiting on #356 to avoid a conflict.

```
AttestedCredentialData.WriteTo(...
 
writer. WriteGuidBigEndian(AaGuid);

```

Since this PR also contains breaking changes, we may be shaping up for a v4 release with Passkey support.

**Comment by abergs** [comment]
Agreed, I think v4 should contain the bits for passkey.

**Review by aseigler — APPROVED** [review]

**Review by abergs — APPROVED** [review]
Reviewed, looks good!

**Pull request merged**